eng/testing/performance/performance-setup.ps1

```diff
@@ -99,6 +100,10 @@ if ($iOSNativeAOT) {
     $Configurations += " iOSStripSymbols=$iOSStripSymbols"
 }
 
+if ($HybridGlobalization) {
+    $Configurations += " hybridGlobalization=$HybridGlobalization"
```

Comment:
Nit: Maybe uppercase `hybridGlobalization` as done in `eng/testing/performance/performance-setup.sh`.